src/main/resources/js/condec.jira.issue.module.js

```diff
@@ -163,6 +169,12 @@
 			if (filterData.endDate >= 0) {
 				secondDatePicker.valueAsDate = new Date(filterData.endDate + 1000);
 			}
+			
+			var linkTypes = conDecAPI.linkTypes;
+			for (var index = 0; index < linkTypes.length; index++) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['index' is already defined.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=4886011)